scripts/install-linux.sh

```
@@ -1,10 +1,35 @@
-uv pip install -e .[fairchem] --no-cache
-uv pip install -e .[orb] --no-cache
-uv pip install -e .[matgl] --no-cache
-uv pip install -e .[test,extra] --no-cache
-uv pip install -e .[mace] --no-cache
+GROUP=${1:-default}
+
+if [ "$GROUP" == "nequip" ]; then
+    TORCH=2.5
+    CUDA=cu124
+    uv pip install torch==${TORCH}.0
+    uv pip install -e .[test,extra,nequip] --no-cache
```

Shellcheck (0.11.0)

[info] Ranges can only match single chars (mentioned due to duplicates).

(SC2102)